packages/gestalt/src/PopoverEducational.js

```diff
@@ -170,35 +170,34 @@ export default function PopoverEducational({
 
   if (!isInExperiment)
     return (
-      <Box zIndex={zIndex} position={zIndex ? 'relative' : undefined}>
```

Comment:
Moving zIndex from this Box to Contents component with position absolute.

<Box zIndex={zIndex} position={zIndex ? 'relative' : undefined}>